guides/doc-Administering_Red_Hat_Satellite/topics/restoring-from-a-full-backup.adoc

```diff
@@ -6,7 +6,7 @@ When the restore process completes, all processes are online, and all databases
 
 .Prerequisites
 * Ensure that you are restoring to the correct instance.
-The {ProjectName} instance must have the same host name, configuration, and be the same major version as the original system.
+The {ProjectName} instance must have the same host name, configuration, and be the same version as the original system.
```

Comment:
Major was certainly correct, but I think for both upstream we can say it must be the same minor version. Thoughts on:
```suggestion
The {ProjectName} instance must have the same host name, configuration, and be the same minor version (X.Y) as the original system.
```

Otherwise I suspect someone might think it needs to be the same patch version and we get questions about that.

Reply:
+1 thank you @ekohl